moveit_planners/ompl/ompl_interface/CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
 set(MOVEIT_LIB_NAME moveit_ompl_interface)
 
+if (APPLE)
+  # needed to bring ode on the linker path exported by homebrew's OMPL
+  link_directories(${OMPL_LIBRARY_DIRS})
```

Comment:
Interesting, I didn't know there were any issues with this. Do we need a vendor package, then? @mamoll any suggestions?